### PR TITLE
roachtest: revive cancel roachtest

### LINKED ISSUE #42103: roachtest: update cancel/tpcc to use long-running queries
SHA: https://github.com/cockroachdb/cockroach/commits/4c52530a33367c58e434111324fcda8c3d73582a

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=cancel/tpcc/distsql/w=10,nodes=3 PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1568299&tab=artifacts#/cancel/tpcc/distsql/w=10,nodes=3

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191101-1568299/cancel/tpcc/distsql/w=10_nodes=3/run_1
	cancel.go:65: query completed before it could be canceled

```

### PR DESCRIPTION
**Pull request by yuzefovich**
**roachtest: rename SQLExec to SQLQueries**

Release note: None

**roachtest: revive cancel roachtest**

`cancel` roachtest has been skipped for a long time now. Its idea is to
run some longer running queries and make sure that they can be canceled
with `CANCEL QUERIES`. Previously, we were using TPCC dataset with 10
warehouses and some arbitrary queries. Over time we have improved
significantly so that the execution took shorter time than we
anticipated which resulted in the test not being able to cancel the
query. This commit switches to using TPCH dataset of Scale Factor 1 and
uses several longer running queries (7, 9, 20, 21) to cancel (each is
taking at least 3s right now).

Fixes: #42103.

Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/test_registry.go [hunk 1]
```diff
@@ -81,8 +81,7 @@ var roachtestOwners = map[Owner]OwnerMetadata{
 	OwnerPartitioning: {SlackRoom: `kv`, Mention: []string{`@cockroachdb/kv`},
 		TriageColumnID: 3550674,
 	},
-	// TODO(jordanlewis): that team doesn't exist any more, fix things up.
-	OwnerSQLExec: {SlackRoom: `sql-execution-team`, Mention: []string{`@cockroachdb/sql-queries`},
+	OwnerSQLQueries: {SlackRoom: `sql-queries`, Mention: []string{`@cockroachdb/sql-queries`},
 		TriageColumnID: 6837155,
```
Comment (rafiss) [inline pkg/cmd/roachtest/test_registry.go:85]: this should be `13549252` 

based on this link: https://github.com/cockroachdb/cockroach/projects/61#column-13549252

**Review by rytaft — APPROVED** [review]
:lgtm:

> Reviewed 8 of 8 files at r1, 2 of 2 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/63415#-:-MY6bPIF9Yq-AwEH2FMX:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @yuzefovich)

**Review by RaduBerinde — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/63415#-:-MYDdPp22_b6lb0IwMZn:bnfp4nl)* status: :shipit: complete! 2 of 0 LGTMs obtained (waiting on @yuzefovich)

**Comment by yuzefovich** [comment]
TFTRs!

bors r+

**Pull request merged**